Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -229,7 +229,7 @@ bool CBoot::SetupWiiMemory(DiscIO::IVolume::ECountry country)
 
 		if (!settingsFileHandle.WriteBytes(gen.GetData(), SettingsHandler::SETTINGS_SIZE))
 		{
-			PanicAlertT("SetupWiiMemory: Cant create setting.txt file");
+			PanicAlertT("SetupWiiMemory: Can't create setting.txt file");
```

Comment:
It will remove the string from all translations. When a translator goes to re-translate it, they will be able to see the old translation as a suggestion. However, since this is a PR for master, it will only become a problem if master strings are uploaded to Transifex before 5.0. I don't know if that's going to be done - it might have bad effects on the stable translations.